src/components/RouteOption/RouteOption.jsx

```diff
@@ -1,17 +1,45 @@
+// import React from "react";
+// import PropTypes from "prop-types";
+// import DockedEbikeRouteOption from "./DockedEbikeRouteOption";
+
+// export default function RouteOption({type, ...props}) {
+//   if (type === DockedEbikeRouteOption.TYPE) {
+//     return DockedEbikeRouteOption(props);
+//   } else {
+//     throw TypeError(`Unknown route option type: '${type}'.`);
+//   }
+// }
+
+// RouteOption.propTypes = {
+//   type: PropTypes.oneOf([
+//     DockedEbikeRouteOption.TYPE,
+//   ]),
+// }
+
+
 import React from "react";
 import PropTypes from "prop-types";
 import DockedEbikeRouteOption from "./DockedEbikeRouteOption";
 
-export default function RouteOption({type, ...props}) {
+export default function RouteOption({ type, isSelected, onClick, ...props }) {
+  const style = {
+    backgroundColor: isSelected ? "blue" : "transparent",
+    cursor: "pointer",
+  };
+
   if (type === DockedEbikeRouteOption.TYPE) {
-    return DockedEbikeRouteOption(props);
+    return (
+      <div onClick={onClick} style={style}>
+        <DockedEbikeRouteOption {...props} />
+      </div>
+    );
   } else {
-    throw TypeError(`Unknown route option type: '${type}'.`);
+    throw new TypeError(`Unknown route option type: '${type}'.`);
   }
 }
 
 RouteOption.propTypes = {
-  type: PropTypes.oneOf([
-    DockedEbikeRouteOption.TYPE,
-  ]),
-}
+  type: PropTypes.oneOf([DockedEbikeRouteOption.TYPE]),
```

Comment:
nitpick - was this autoformatted? I prefer to have it on separate lines with a trailing comma as it was before. Because when you do it like this, when we add a new element to the array in the future, that will be a one-line isolated diff. makes it easier to search through line-by-line git history (aka git blame)

Reply:
nevermind - it seems as if [prettier](https://prettier.io/) also formats it like this. will keep your formatting